sendemail.py

```diff
@@ -0,0 +1,16 @@
+from apscheduler.schedulers.blocking import BlockingScheduler
```

Comment:
Is `apscheduler` part of the default Python installation? Otherwise need to edit `requirements.txt`, remember!